File: demo/src/main/java/pyxis/uzuki/live/viewtransformer/demo/java/EditTextActivity.java
package pyxis.uzuki.live.viewtransformer.demo.java;

import android.graphics.Color;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.widget.EditText;

import org.jetbrains.annotations.NotNull;

import pyxis.uzuki.live.viewtransformer.ConsumerType;
import pyxis.uzuki.live.viewtransformer.ViewTransformer;
import pyxis.uzuki.live.viewtransformer.consumer.EditTextConsumer;
import pyxis.uzuki.live.viewtransformer.demo.BaseActivity;
import pyxis.uzuki.live.viewtransformer.demo.R;

/**
 * TextViewChanger
 * Class: TextViewActivity
 * Created by Pyxis on 2017-12-19.
 * <p>
 * Description:
 */

public class EditTextActivity extends BaseActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ViewTransformer.clear();
        ViewTransformer.add(ConsumerType.TypeEditText, (EditTextConsumer<EditText>) editText -> editText.setTextColor(Color.BLUE));
        setContentView(ViewTransformer.transform(R.layout.activity_edittext, this));
        }
        }

File: demo/src/main/java/pyxis/uzuki/live/viewtransformer/demo/kotlin/EditTextActivity.kt
package pyxis.uzuki.live.viewtransformer.demo.kotlin

import android.graphics.Color
import android.os.Bundle
import android.widget.EditText
import android.widget.TextView
import pyxis.uzuki.live.viewtransformer.ConsumerType
import pyxis.uzuki.live.viewtransformer.ViewTransformer
import pyxis.uzuki.live.viewtransformer.consumer.EditTextConsumer
import pyxis.uzuki.live.viewtransformer.consumer.TextViewConsumer
import pyxis.uzuki.live.viewtransformer.demo.BaseActivity
import pyxis.uzuki.live.viewtransformer.demo.R
import pyxis.uzuki.live.viewtransformer.preset.TextSizeChanger

/**
 * TextViewChanger
 * Class: TextViewActivity
 * Created by Pyxis on 2017-12-19.
 *
 * Description:
 */
class EditTextActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        ViewTransformer.add(ConsumerType.TypeEditText, EditTextConsumer<EditText> { editText ->
            editText.setTextColor(Color.BLUE)
        })
        setContentView(ViewTransformer.transform(R.layout.activity_edittext, this))
    }
}